elifetools/tests/test_xmlio.py

```diff
@@ -165,6 +165,14 @@ def test_rewrite_subject_group(self, xml, subjects, subject_group_type, overwrit
             rough_string = ElementTree.tostring(root, encoding='unicode')
         self.assertEqual(rough_string, xml_expected)
 
+    def test_reparsed_tag(self):
+        """test parsing a string into xml.dom"""
+        tag_name = 'root'
+        tag_string = 'Test &amp; <i class="a">a</i> <mml:math><mml:mi>K</mml:mi></mml:math>.'
```

Comment:
it's a complex case (XML `&` entities, nested tags with namespaces), but it matches the fixture

Reply:
These are the sorts of things it should be able to reparse so I made the test reflect the complexity 👍